test/__snapshots__/bindings.test.ts.snap

```diff
@@ -123,6 +332,9 @@ public interface Name extends software.amazon.jsii.JsiiSerializable {
         private java.lang.String first;
         private java.lang.String last;
         private java.lang.String middle;
+        private org.myorg.NameObjectWithPropertiesAndBooleanAdditionalProperties objectWithPropertiesAndBooleanAdditionalProperties;
```

Comment:
This should have been somehow an intersection with `java.util.Map<java.lang.String, java.lang.Object>`